test.js

```diff
@@ -129,7 +139,7 @@ it('normal throw', function() {
     'throw new Error("test");'
   ], [
     'Error: test',
-    /^    at Object\.exports\.test \(.*\/line1\.js:1001:101\)$/
+    /^    at Object\.exports\.test \((?:.*\/)?line1\.js:1001:101\)$/
```

Comment:
It's all down to https://github.com/mozilla/source-map/commit/d281271b106e3e4cc5e06eb8f1589c02cd0d5219 which means that, for example `util.join('.', 'foo.js')` is now `foo.js` rather than `./foo.js` so yes, basically it's down to source-map changing so that it no longer gives us a `./` prefix.